src/EnergyPlus/SZVAVModel.cc

```diff
@@ -977,7 +977,7 @@ namespace SZVAVModel {
                         bool const &HXUnitOn,
```

Comment:
I thought we had gotten rid of these `bool const &` and `int const &` parameters.